### DISCUSSION TITLE
Add an indicator for when chatterino is offline/hasnt received a message for x seconds

### DISCUSSION THREAD

**Discussion by 12932** [Chatterino/chatterino2#4155] [Ideas]
Currently, theres no way to tell if your internet is broken or if chatterino is connected or not

I feel like there should be an indicator somewhere, that shows Chatterino hasnt received a message for x seconds , maybe just above the chat input field

**Comment by LosFarmosCTL** [comment]
Chatterino disconnecting should send a message in chat already, not sure what you mean by not being able to tell if your internet has disconnected.
